docs/product-getting-started-guide/src/main/asciidoc/data-model-proc.adoc

```diff
@@ -1,9 +1,9 @@
 [id='_defining_a_data_model']
-= Creating the *Mortgage_Process* data model
+= Creating the *Mortgage Process* data model
 
-The goal of this chapter is to introduce you to the *Mortgage_Process* data model. Data models are comprised of data objects. A data object is a custom complex data type that contains data fields such as Name, Address, and Credit Rating.
+The goal of this chapter is to introduce you to the *Mortgage Process* data model. Data models are comprised of data objects. A data object is a custom complex data type that contains data fields such as Name, Address, and Credit Rating.
 
-The *Mortgage_Process* data model is comprised of four data objects:
+The *Mortgage Process* data model is comprised of four data objects:
```

Comment:
I'm not gonna tell you again, lady :) "is comprised of" needs to be either "is composed of" or "comprises". Not an opinion, a grammatical fact :) Search all instances of "comprised of".

Reply:
pfffffffffffffffffffffffffffffff Sir